### ISSUE TITLE
A11y_AzureDataStudioSQLMigration_MigratetoAzureSQL_Databasebackup-networkshare_DeveloperTool:There is no label defined for the resource group combo box.

### ISSUE THREAD

**Issue by tripathiakhil429** [microsoft/azuredatastudio#15694]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yWCAG2.1;#A11yTCS;#A11ySev2;#SH_AZUREDATASTUDIOSQLMigration_June2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG2.5.3;#AINotInScope;#DevTool

### Environment Details:
Application Name: SQL Azure Data Studio
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure SQL Migration.
2. Connect to any sql server.
3. Tab till Azure SQL migration Tab and press enter.
4. Tab till Migrate to azure sql and press enter.
5. Add azure account and tab till next button and press enter.
6. Select SQL server on azure virtual machine radio button and fill all the required fields and tab till next button and press enter.
7. Check Online migration radio button and tab till next button and press enter.
8. Check "My database are on a network share" radio button.
9. Tab till resource group combo-box.
10. Open Developer Tool and hover over to the combo box.

### Actual:
There is no label defined for the resource group combo box.

### Expected:
There should be a proper label "resource group" defined for the combo box.
### 
### User Impact:
Screen reader users will not know the purpose of the combo box.

**Comment by tripathiakhil429** [comment]
closing as duplicate.

**Comment by 12-shweta** [comment]
**Github Tags:**#Closed;
